node/tests/README.md

```diff
@@ -0,0 +1,40 @@
+# Swamp: In-Memory Test Tool
+
+Swamp is a testing tool that creates an environment for deploying `celestia-node` and testing instances against each other. 
+While the swamp takes care of setting up networking and initial configuration of node types, the user can focus on tailoring test scenarios
+
+## Usage
+### Creating a swamp
+Calling `NewSwamp` will return you a new constructed swamp with the mocknet. That function looks like this:
+
+```go
+swamp := NewSwamp(t)
+```
+The first parameter of the swarm constructor is the `testing.T`
+
+### Constructing celestia nodes
+You can construct any celestia `bridge/full/light` nodes using swamp and be rest assured that they will be linked between each other.
+<i>Note: Linking nodes does not mean that you have connected them</i>
+
+```go
+// you can add any options during initialisation in the signature
```

Comment:
For consistency, use American English everywhere, e.g. initialization.

Reply:
Ok